fabric-item-group-api-v1/src/main/java/net/fabricmc/fabric/api/itemgroup/v1/IdentifiableItemGroup.java

```diff
@@ -24,7 +24,7 @@
  */
 public interface IdentifiableItemGroup {
 	/**
-	 * {@return the unique identifier of this {@link ItemGroup}}
+	 * {@return the unique identifier of this {@link ItemGroup}}.
```

Comment:
Does this lead to an extra dot at the end?

Reply:
Checkstyle fails without it.

Reply:
I know but that's not what I was asking 😛

Reply:
I should learn to read one day...

Just checked, no it doesnt.

Reply:
TIL, then it's all good.